dspace-server-webapp/src/main/java/org/dspace/app/rest/utils/ApplicationConfig.java

```diff
@@ -13,17 +13,29 @@
 import org.springframework.context.annotation.Configuration;
 
 /**
- * This class provides extra configuration for our Spring Boot Application
+ * This class provides extra configuration for our Spring Boot Application.
  * <p>
- * NOTE: @ComponentScan on "org.dspace.app.configuration" provides a way for other DSpace modules or plugins
- * to "inject" their own Spring configurations / subpaths into our Spring Boot webapp.
+ * NOTE: @ComponentScan on "org.dspace.app.configuration" provides a way for
+ * other DSpace modules or plugins to "inject" their own Spring configurations /
+ * subpaths into our Spring Boot webapp.
  *
  * @author Andrea Bollini (andrea.bollini at 4science.it)
  * @author Tim Donohue
  */
 @Configuration
-@ComponentScan( {"org.dspace.app.rest.converter", "org.dspace.app.rest.repository", "org.dspace.app.rest.utils",
-        "org.dspace.app.configuration", "org.dspace.iiif", "org.dspace.app.iiif"})
+// Component scanning ignores any parent {@code ApplicationContext}s, so any
+// bean which is in the scope of both will be duplicated.  dspace-services makes
+// its context the parent of this one.  If a bean is explicitly configured in
+// the parent, it won't be so configured in this context and you may have
+// trouble.  IOW be careful what you add here.
```

Comment:
I'd recommend spelling out "IOW" to be "In other words".  It's helpful for those who are not familiar with the abbreviation or not native English speakers.

Reply:
"IOW" doesn't really add any meaning here, so I think I'll just remove it.